extensions/data-plane/data-plane-framework/src/main/java/org/eclipse/dataspaceconnector/dataplane/framework/manager/DataPlaneManagerImpl.java

```diff
@@ -43,11 +46,13 @@
     private int workers = 1;
     private long waitTimeout = 100;
 
+    private List<TransferService> transferServices = new ArrayList<>();
     private PipelineService pipelineService;
```

Comment:
Is the pipeline service still required here? Can we remove the methods:
```
@Override
    public CompletableFuture<TransferResult> transfer(DataSource source, DataFlowRequest request) {
        return pipelineService.transfer(source, request);
    }

    @Override
    public CompletableFuture<TransferResult> transfer(DataSink sink, DataFlowRequest request) {
        return pipelineService.transfer(sink, request);
    }
```?

Reply:
Yes per Paul these are required for the "Proxy" function

Reply:
Do you know what this proxy function is all about?